test/validations.test.js

```diff
@@ -322,6 +322,43 @@ describe('validations', function() {
       });
     });
 
+    it('should pass options to custom synch', function(done) {
+      delete User.validations;
+      User.validate('name', function(err, options) {
+        if (options.options !== 'someValue') err();
+      });
+      User.create({name: 'Valid'}, {options: 'someValue'}, function(e, d) {
+        d.updateAttribute('name', null, {options: 'otherValue'}, function(e) {
+          should.exist(e);
+          e.should.be.instanceOf(Error);
+          e.should.be.instanceOf(ValidationError);
+          d.updateAttribute('name', 'Vasiliy', {options: 'someValue'}, function(e) {
+            should.not.exist(e);
+            done();
+          });
+        });
+      });
+    });
+
+    it('should pass options to custom async', function(done) {
+      delete User.validations;
+      User.validateAsync('name', function(err, options, done) {
+        if (options.options !== 'someValue') err();
+        process.nextTick(function() { done(); });
+      });
+      User.create({name: 'Valid'}, {options: 'someValue'}, function(e, d) {
+        d.updateAttribute('name', null, {options: 'otherValue'}, function(e) {
+          should.exist(e);
+          e.should.be.instanceOf(Error);
+          e.should.be.instanceOf(ValidationError);
+          d.updateAttribute('name', 'Vasiliy', {options: 'someValue'}, function(e) {
+            should.not.exist(e);
```

Comment:
```js
if (err) return done(err);
```

Your way discard details and the stack trace from the original error and produces a stack trace pointing to this line, which is rather unhelpful.